core/cc/android/crash_handler.cpp

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
If 99% of this file is identical between Android, macOS and Linux, I'd suggest putting the bulk in the posix directory, and use `static const char*` for for the file location (which seems to be the only thing I can see that's different). You can then define that `static const char*` in these OS specific directories.

Reply:
Android and Linux are indeed nearly identical. MacOS though has different function signatures for the ExceptionHandler constructor and the handleCrash static callback, it takes a MinidumpDescriptor instead of just a plain path.